cypress/e2e/loans/renewal-failure.cy.js

```diff
@@ -156,7 +156,6 @@ describe('Renewal', () => {
     CheckinActions.checkinItemViaApi({
```

Comment:
please fix issue: 'moment' is defined but never used. Allowed unused vars must match /React/u